### PR TITLE
CLDR-14140 Distinguish metric from non-metric

### PR DESCRIPTION
**Pull request by macchiati**
Note: this is informative information and shouldn't have an effect on ICU.

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14140
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java [hunk 1]
```diff
@@ -929,37 +933,78 @@ public void TestContinuationOrder() {
...
-                    || ukSystemPattern.matcher(inputFactor).find());
-
-            Collection<String> systems = toSystems.get(unit);
-            if (systems == null) {
-                systems = Collections.emptySet();
-            }
-            if (!assertEquals(unit + ": US? (" + inputFactor + ")", usSystem, systems.contains("ussystem"))) {
-                int debug = 0;
+    public void TestSystems() {
+//        Map<String, TargetInfo> data = converter.getInternalConversionData();
```
Comment (hugovdm) [inline tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java:950]: Is it useful to keep the commented-out code in this function?

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java [hunk 2]
```diff
@@ -929,37 +933,78 @@ public void TestContinuationOrder() {
...
+        Output<String> metricUnit = new Output<>();
+        Multimap<Set<UnitSystem>, R3<String, ConversionInfo, String>> systemsToUnits = TreeMultimap.create(Comparators.lexicographical(Ordering.natural()), Ordering.natural());
+        for (String longUnit : VALID_REGULAR_UNITS) {
+            String unit = Units.getShort(longUnit);
+            if (unit.equals("generic")) {
+                continue;
             }
-            if (!assertEquals(unit + ": UK? (" + inputFactor + ")", ukSystem, systems.contains("uksystem"))) {
+            if (unit.contentEquals("centiliter")) {
                 int debug = 0;
```
Comment (hugovdm) [inline tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java:959]: I'm curious what this "centiliter? then debug=0" is about? (I see code style isn't to have lots of explanatory comments though.)

**Review by hugovdm — APPROVED** [review]

File: tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java (hunk 2 above)
Comment (macchiati) [reply tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestUnits.java:959]: Those are just debug statements; there is a separate ticket to clean those up.

**Comment by macchiati** [comment]
Tom, can you review this? I can't reach Steven (and Hugo isn't in the CLDR org, so I need a different review).

**Review by btangmu — APPROVED** [review]

**Review by srl295 — COMMENTED** [review]
post merge LGTM

**Pull request merged**